lib/slather/coverage_file.rb

```diff
@@ -14,10 +14,13 @@ def source_file_pathname
         # TODO: Handle Swift
         path = nil
         if project.source_directory
-          path = Dir["#{project.source_directory}/**/#{base_filename}.m"].first
+          path = Dir["#{project.source_directory}/**/#{base_filename}.{m,mm,cpp,hpp,swift}"].first
```

Comment:
I'm not sure if Swift belongs here since there will be no gcov files for Swift ([rdar://17450338](http://openradar.appspot.com/radar?id=6457429508227072)).